tools/osbuilder/node-builder/azure-linux/Makefile

```diff
@@ -23,21 +21,49 @@ all: package uvm
 all-confpods: package-confpods uvm-confpods
```

Comment:
@ms-mahuber Right now this means the all-* targets build both release and debug - this slows down iterating during dev by 2x.

I propose we create all-debug and all-confpods-debug, which would only build in debug mode. This restores the current behavior/speed in the dev flow.

Reply:
I have added instructions in the README.md for azurelinux to work around this by calling the make uvm-[confpods]-[debug/release] targets. Furthermore, uvm_install.sh no longer expects both sets of artifacts to be there, can tolerate debug OR release OR both. Install still fails if both are missing.